src/microbe_stage/editor/MicrobeEditorReportComponent.cs

```diff
@@ -157,6 +158,8 @@ public void UpdatePatchDetails(Patch currentOrSelectedPatch, Patch? selectedPatc
 
         UpdateReportTabPatchName(currentOrSelectedPatch);
 
+        UpdateReportTabPatchSelector();
```

Comment:
Shouldn't this only be called if the player actually moves? Right now I'm not 100% certain but I think that UpdatePatchDetails can get called even when the player is just clicking around patches to look at their details.